{{cookiecutter.repostory_name}}/envs/prod/.vuln.env.template

```diff
@@ -0,0 +1,5 @@
+ENV=prod
+DD_URL=
+DD_API_KEY=
+DD_PRODUCT=
```

Comment:
this is not explained, there should be a mention of the convention: product=client_server_group

Reply:
That's a good point, but I was thinking this should be somewhere in our internal handbook in the relevant section about vulnerabilities management. Here we just referring to whatever was created in DefectDojo. Wdyt?

Reply:
I'm not sure, leaving it in another place is gonna be gelpful, people will just forget about it

Reply:
Added some clarifying comments to the `.env` file